lib/stdlib/src/queue.erl

```diff
@@ -706,7 +706,7 @@ r2f([_]=R) ->
 r2f([X,Y]) ->
     {[X],[Y]};
 r2f(List) ->
-    {FF,RR} = lists:split(length(List) div 2 + 1, List),
+    {FF,RR} = lists:split(length(List) div 2, List),
     {FF,lists:reverse(RR, [])}.
```

Comment:
Can you, while you are at it, swap the variable names `FF` and `RR` here? Now I think they are confusing...

Reply:
Thanks, much better!